packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -15,6 +14,8 @@ export interface DataListProps extends React.HTMLProps<HTMLUListElement> {
   onSelectDataListItem?: (id: string) => void;
   /* Id of DataList item currently selected */
   selectedDataListItemId?: string;
+  /** Flag indicating if DataList should have compact styling */
+  isCompact?: boolean;
```

Comment:
looks like a feature addition to me, perhaps better in a separate PR?

Reply:
This change comes from merging `master` into `v4`.